support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx

```diff
@@ -156,56 +158,59 @@ const benefitsPrefixPlus = css`
 `;
 
 const discountSummaryCopy = (
-	currency: string,
-	planCost: TierPlanCosts,
+	currency: Currency,
 	promoCount: number,
+	price: number,
+	promotion: Promotion,
+	paymentFrequency: RegularContributionType,
 ) => {
-	/* EXAMPLE:
-  £6.5/month for 6 months, then £10/month
-  £173/year for the first year, then £275/year
-  */
-	if (planCost.discount) {
-		const period = planCost.discount.duration.period;
-		const duration = planCost.discount.duration.value;
-		const singleYear =
-			period === 'ANNUAL' && duration === 1 ? ' the first ' : '';
-		const promoPrice = planCost.discount.price;
-		const promoPriceRounded =
-			promoPrice % 1 === 0 ? promoPrice : promoPrice.toFixed(2);
+	/**
+	 * EXAMPLE:
+	 * - £6.5/month for 6 months, then £10/month
+	 * - £173/year for the first year, then £275/year
+	 */
+	const durationMonths = promotion.discount?.durationMonths ?? 0;
+	const formattedPrice = simpleFormatAmount(currency, price);
+	const formattedPromotionPrice = simpleFormatAmount(
+		currency,
+		promotion.discountedPrice ?? 0,
+	);
+	const period = paymentFrequency === 'ANNUAL' ? 'year' : 'month';
+	const duration =
+		paymentFrequency === 'ANNUAL' ? durationMonths / 12 : durationMonths;
 
-		return `${currency}${promoPriceRounded}/${
-			recurringContributionPeriodMap[planCost.discount.duration.period]
-		} for ${duration > 1 ? duration : singleYear} ${
-			recurringContributionPeriodMap[period]
-		}${duration > 1 ? 's' : ''}, then ${currency}${planCost.price}/${
-			recurringContributionPeriodMap[planCost.discount.duration.period]
-		}${'*'.repeat(promoCount)}`;
-	}
+	return `${formattedPromotionPrice}/${period} for ${
+		period === 'year' ? ' the first ' : ''
+	} ${duration} ${period}${
```

Comment:
Think this should read  `${duration}` should read `${duration > 1 ? duration : ''}`

Reply:
Then it will read `$378/year for the first year, then $480/year*` instead of `$378/year for the first 1 year, then $480/year*`

Reply:
```suggestion
	} ${duration > 1 ? duration : ''} ${period}${
```